code/API_definitions/location-subscriptions.yaml

```diff
@@ -0,0 +1,590 @@
+openapi: 3.0.3
+info:
+  title: Device location subscription event API
+  description: Service Enabling Network Function API which create, retrieve and delete subscriptions for events related to the location of a device
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0-wip
+externalDocs:
+  description: Product documentation at Camara
+  url: https://github.com/camaraproject/
+security:
+  - oAuth2ClientCredentials: []
+  
+servers:
+  - url: '{apiRoot}/{basePath}'
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      basePath:
+        default: location-subscription/v0
+        description: Base path for the device location subscription event API
+paths:
+  /event-subscriptions:
+    post:
+      tags:
+        - Location Event Subscription
+      summary: 'Create a location event subscription for a device'
+      operationId: createEventSubscription
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateEventSubscription'
+        required: true
+      callbacks:
+        event-notifications:
+          '{$request.body#/webhook/notificationUrl}':
+            post:
+              tags:
+                - Notification Callback
+              summary: "Event Subscription notification callback"
+              operationId: postEventNotification
+              requestBody:
+                required: true
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/EventNotifications"
+              responses:
+                '204':
+                  description: Successful event notification
+                '400':
+                  $ref: '#/components/responses/Generic400'
+                '401':
+                  $ref: '#/components/responses/Generic401'
+                '403':
+                  $ref: '#/components/responses/Generic403'
+                '500':
+                  $ref: '#/components/responses/Generic500'
+                '503':
+                  $ref: '#/components/responses/Generic503'
```

Comment:
we need some security for the notification, you can copy it from qod

security:
                - {}
                - notificationsBearerAuth: []